lib/avtp_pipeline/platform/Linux/openavb_tasks.h

```diff
@@ -32,7 +32,13 @@ Complete license and copyright information can be found at
 #ifndef _EAVBTASKS_H
 #define _EAVBTASKS_H
 
-#define THREAD_STACK_SIZE 									65536
+#if !defined(PTHREAD_STACK_MIN)
+#error "PTHREAD_STACK_MIN variable not defined"
```

Comment:
@msutar, on Linux PTHREAD_STACK_MIN is defined in limits.h. This header file is not included.